lib/phoenix_mtm/helpers.ex

```diff
@@ -36,8 +36,16 @@ defmodule PhoenixMTM.Helpers do
     selected = Keyword.get(opts, :selected, [])
     input_opts = Keyword.get(opts, :input_opts, [])
     label_opts = Keyword.get(opts, :label_opts, [])
+    wrapper = Keyword.get(opts, :wrapper, &PhoenixMTM.Wrappers.unwrapped/6)
 
-    inputs = Enum.map(collection, fn {label, value} ->
+    # TODO: Eventually deprecate this option in favour of passing in custom wrapper
```

Comment:
Maybe log a warning about the deprecation so users of the library have ample time to update their code?